src/impy/models/dpmjetIII.py

```diff
@@ -170,7 +170,9 @@ def _generate(self):
             k.p1.Z or 0,
             k.p2.A or 1,
             k.p2.Z or 0,
-            self._lib.idt_icihad(k.p1),
```

Comment:
@afedynitch The tests are now failing again on CI since this change. It looks like you regenerated all the references for Dpmjet and phojet, so it should be consistent. Any idea why this is failing?

Reply:
This is indeed weird. The tests pass on my AMD and M1 Mac. I generated the files on the AMD under Linux. The only thing I can imagine doing at the moment is to generate them again on the Mac and try again. 

Ultimately, we/I/Anton need to figure out what causes the RND sequence divergences.

Reply:
The RND sequences themselves are consistent between platforms and machines, unless I missed something, the tentative conclusion was that the differences arise from different stdlib implementations on different platforms/compiler versions. I think we have to live with this and use a very lenient threshold for the p-value in the test.